### ISSUE TITLE
how to (un)check checkbox only with link

### ISSUE THREAD

**Issue by markushub** [stefan-niedermann/nextcloud-notes#1328]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

**Describe the bug**

Try to check or uncheck in Preview a checkbox but allways link to website

**To Reproduce**

Create new note
```

- [ ] https://github.com/stefan-niedermann/nextcloud-notes
- [x] https://github.com/stefan-niedermann/nextcloud-notes/issues

```

Click to check checkbox

*Expected behavior**

Checkbox will be checked

**Screenshots**

**Smartphone (please complete the following information):**
 - Nextcloud Notes-Version (android app): v3.4.11
 - F-Droid
 - Android-Version: 11
 - Device: potter

**Server**
 - Nextcloud version: 22.1
 - Nextcloud Notes version (server app): 

**Stacktrace**

**Comment by stefan-niedermann** [comment]
I agree that it should still be possible to toggle such kind of checkboxes by clicking on the checkbox directly. This is an issue caused by our recent efforts to make links in checkboxes clickable at all.

**Comment by tysino** [comment]
I'd also appreciate this feature. But maybe not in preview mode, because I think changes in the note would be unexpected there. Maybe it is possible in editing mode to view a checkbox which can be toggled after the character sequence ```- [ ]``` (like the bullet points in lists which are already shown).
